utils/debug/runtime.go

```diff
@@ -0,0 +1,16 @@
+package debug
+
+import (
+	"runtime/metrics"
+)
+
+// GetHeapAllocsBytes returns the value of /gc/heap/allocs:bytes.
+func GetHeapAllocsBytes() uint64 {
+	sample := []metrics.Sample{{Name: "/gc/heap/allocs:bytes"}}
```

Comment:
```suggestion
	// https://pkg.go.dev/runtime/metrics#hdr-Supported_metrics
	sample := []metrics.Sample{{Name: "/gc/heap/allocs:bytes"}}
```

Better to add the reference for the "magic string"